### DISCUSSION TITLE
Error Deploy AdonisJs application with Postgres and Redis on DigitalOcean

### DISCUSSION THREAD

**Discussion by ItsJuniorDias** [community/community#19734] [Projects and Issues] [answered]
**Problem occurs in the migration generation process**

<img width="1115" alt="Captura de Tela 2022-06-29 às 22 24 54" src="https://user-images.githubusercontent.com/50254416/176572851-42a1f9fe-9646-415c-8fd3-08c34abd30e4.png">

**Comment by rethink-alexandrejunior** [comment, marked as answer]
you can't run the adonis command because you don't have the cli installed on your cluster
